draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -427,8 +443,8 @@ A zone state reset may be performed by a change of the member node's name (see (
 # Implementation and operational Notes {#implementationnotes}
 
 Although any valid domain name can be used for the catalog name $CATZ, it is
-RECOMMENDED to use either a domain name owned by the catalog producer, or to
-use a name under a suitable Special-Use Domain Name [@!RFC6761].
+RECOMMENDED to use either a domain name controlled by the catalog producer, or
+to use a name under a suitable name such as "invalid." [@!RFC6761].
```

Comment:
> Why replace "owned" by "controlled"?

Because I don't think you can "own" a name under `invalid.`. What matters is that the producer controls it in the DNS context at hand. (Similarly, it's sufficient for the ACME protocol that a certificate requestor controls the name.)

> Also, reintroducing Special-Use domain name (but keeping the suggestion for "invalid.") to allow for future domain names under ".alt"

My impression was that Warren was not happy about mentioning special-use names (`what is a "suitable Special-Use Domain Name"?` -- note that his question was not simply what "suitable" means). I think this is because the ".alt" discussion has sparked uncertainty about the future of RFC 6761.

I therefore thought that any name deemed suitable in the local context can serve for this, notwithstanding use of future domain names under ".alt", and only vaguely referring to the RFC via the reference.

These were my attempts to "water down" the specificity of normative statements in a pragmatic way so that the purpose is served, while forestalling potential discussion about the role of RFC 6761 names. -- However, I have no stakes at all for the actual wording, and am happy to concede to whatever the group deems best.

Reply:
> > Why replace "owned" by "controlled"?
> 
> Because I don't think you can "own" a name under `invalid.`. 

No, but is says "or". So either "owned" or "something under invalid."

> What matters is that the producer controls it in the DNS context at hand. (Similarly, it's sufficient for the ACME protocol that a certificate requestor controls the name.)

Ack. But part of the intention of the paragraph was to prevent hijacking domain name space (as we have done in our testbeds b.t.w. with the "catalog." tld.). Therefore the emphasis on either "owned" or "specially reserved"

> > Also, reintroducing Special-Use domain name (but keeping the suggestion for "invalid.") to allow for future domain names under ".alt"
> 
> My impression was that Warren was not happy about mentioning special-use names (`what is a "suitable Special-Use Domain Name"?` -- note that his question was not simply what "suitable" means). I think this is because the ".alt" discussion has sparked uncertainty about the future of RFC 6761.

> I therefore thought that any name deemed suitable in the local context can serve for this, notwithstanding use of future domain names under ".alt", and only vaguely referring to the RFC via the reference.
> 
> These were my attempts to "water down" the specificity of normative statements in a pragmatic way so that the purpose is served, while forestalling potential discussion about the role of RFC 6761 names. -- However, I have no stakes at all for the actual wording, and am happy to concede to whatever the group deems best.